README.md

```diff
@@ -1,13 +1,19 @@
-# Project Name
+# Guess Who - Technigo Bootcamp Week 5
```

Comment:
**README file**

I know JavaScript can be tricky and this week indeed challenged us all! 😅

As a general comment on the README file moving forward, I would suggest that you write a bit more about how you approached the problem and what tools and techniques you used to solve it. Let the person reading it know more about how you planned the project and how you solved the different parts of it.

Reply:
YES! Make it shine like a diamond 💎 This is the cover of your repo, the invitation to your magical code 🦄